eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,25 @@
+parameters: 
+  TargetUserVariable: 'notspecified'
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+
+steps:
+  - task: DotNetCoreCLI@2
```

Comment:
I suspect using the DotNetCoreCLR task isn't buying you anything over just pwsh when calling a custom command like we are.

Reply:
Put it to pwsh